features/call/src/test/kotlin/io/element/android/features/call/utils/CallIntentDataParserTest.kt

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package io.element.android.features.call
+package io.element.android.features.call.utils
 
-import com.google.common.truth.Truth.assertThat
+import com.google.common.truth.Truth
```

Comment:
I prefer the previous import, the code looks cleaner. But happy with this until we decide and add a Konsist test for this :)